all-spark-notebook/README.md

```diff
@@ -243,11 +244,16 @@ source deactivate
 The commands `ipython`, `python`, `pip`, `easy_install`, and `conda` (among others) are available in both environments.
 
 
-## JupyterHub
+## JupyterHub <a id="JupyterHub"></a>
```

Comment:
Pretty sure you can just link to `#JupyterHub` in the git blob view without the `<a>`